setup.py

```diff
@@ -438,6 +439,33 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     "pandas/_libs/tslibs/src/datetime/np_datetime_strings.h",
 ]
 
+import sysconfig
+
+ext_libraries = [
+    [
+        "np_datetime",
+        {
+            "sources": [
+                "pandas/_libs/tslibs/src/datetime/np_datetime.c",
+                "pandas/_libs/tslibs/src/datetime/np_datetime_strings.c",
+            ],
+            "include_dirs": [
+                "pandas/_libs/tslibs/src/datetime",
+                sysconfig.get_path("include"),
```

Comment:
Do we need these `get_path("include")` calls? I think the compiler should handle that natively